src/expressions/functions/builtInFunctions.node.ts

```diff
@@ -86,6 +93,22 @@ const fnName: FunctionDefinitionType = function(
 	});
 };
 
+const fnHasChildren: FunctionDefinitionType = function(
+	_dynamicContext,
+	_executionParameters,
+	_staticContext,
+	nodeSequence: ISequence
+) {
+	return zipSingleton([nodeSequence], ([nodeValue]: (Value | null)[]) => {
+		const node: ConcreteNode = nodeValue ? nodeValue.value : null;
+
+		if (node && node.firstChild) {
```

Comment:
I believe we should use the `executionParameters.domFacade` to retrieve the child node.
Note: `executionParameters` is now `_executionParameters`